client/java-armeria/src/main/java/com/linecorp/centraldogma/client/armeria/AbstractArmeriaCentralDogmaBuilder.java

```diff
@@ -129,10 +142,12 @@ protected final EndpointGroup endpointGroup() throws UnknownHostException {
         final List<EndpointGroup> groups = new ArrayList<>();
         for (final InetSocketAddress addr : hosts) {
             if (addr.isUnresolved()) {
-                groups.add(DnsAddressEndpointGroup.builder(addr.getHostString())
-                                                  .eventLoop(clientFactory.eventLoopGroup().next())
-                                                  .port(addr.getPort())
-                                                  .build());
+                final DnsAddressEndpointGroupBuilder dnsAddressEndpointGroup = DnsAddressEndpointGroup
+                        .builder(addr.getHostString())
+                        .eventLoop(clientFactory.eventLoopGroup().next())
+                        .port(addr.getPort());
```

Comment:
Could you call this `.port(addr.getPort())` after
`dnsAddressEndpointGroupConfigurator.configure(dnsAddressEndpointGroup);` is called?
Because there's a chance that a user sets the port by mistake.

Reply:
Changed. Meanwhile, as a "user", let me protest against babysitting : when I make mistakes I can fix them, but when the library designer prevents me from doing what he thinks is a mistake I can only weep...